share/package.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env sh
+
+if [ $# -eq 0 ]
+  then
+    echo "Path to package need to be specify"
+	exit 1
+fi
+
+if [[ `mkdir -p "work/gradle/wrapper"` -ne 0 ]];
+	then
+        echo "Cannot create work folder"
```

Comment:
more descriptive message required

`Cannot create a work folder required to install desired package on AEM."